src/index.js

```diff
@@ -38,6 +38,20 @@ import './index.css'
 
 import { Router, Route, IndexRoute, hashHistory } from 'react-router'
 
+import Raven from 'raven-js'
+
+Raven.config('https://2f0fec57a8da462db1301ea38ef0790b@sentry.io/1364120', {
+  collectWindowErrors: true,
+  fetchContext: true,
+  linesOfContext: 40
+}).install()
+
+var originalConsoleError = console.error
```

Comment:
Use `const` instead of `var` to be ES6 consistent

Reply:
Please pull and do it off. I'm unable to fetch time.